lib/routers/establishment/projects.js

```diff
@@ -97,27 +100,42 @@ router.param('id', (req, res, next, id) => {
         .select('id', 'grantedAt', 'submittedAt', 'createdAt')
         .where({ projectId: project.id })
         .orderBy('createdAt', 'desc')
-        .then(versions => {
-          const granted = versions.find(v => v.grantedAt) || null;
-          if (granted) {
-            // if a granted version exists, we're only interested in drafts created after
-            versions = versions.filter(v => v.createdAt > granted.createdAt);
-          }
-          // only attach most recent draft
-          const draft = versions[0] || null;
-          return { granted, draft };
-        })
         .then(versions => {
           req.project = {
             ...project,
-            ...versions
+            versions
           };
           next();
         });
     })
     .catch(next);
 });
 
+const canFork = (req, res, next) => {
+  const { ProjectVersion } = req.models;
+  ProjectVersion.query()
+    .select('id', 'grantedAt', 'submittedAt', 'createdAt')
+    .where({ projectId: req.project.id })
+    .orderBy('createdAt', 'desc')
+    .then(versions => versions[0])
```

Comment:
Are the versions not already available on `req.project.versions`? Is there a need to go back to the database here?

Reply:
good spot 👍